Gemfile

```diff
@@ -58,6 +59,5 @@ group :development, :test do
   gem 'spring'
   gem 'capybara'
   gem 'mocha'
-  gem 'pry'
   gem 'launchy'
 end
```

Comment:
Good job on catching the duplicate pry :smile: 
How about going ahead and setting versions for all the :gem: s now?